src/Dangl.SchneidControl/Services/DataLoggingScheduler.cs

```diff
@@ -17,12 +22,38 @@ public Task StartAsync(CancellationToken cancellationToken)
             Task.Run(async () =>
             {
                 DateTime lastStart;
+                DateTime? lastEmailSent = null;
                 while (_isRunning)
                 {
                     lastStart = DateTime.UtcNow;
                     using var scope = _serviceProvider.CreateScope();
                     var dataLoggingService = scope.ServiceProvider.GetRequiredService<IDataLoggingService>();
-                    await dataLoggingService.ReadAndSaveValuesAsync();
+                    var valuesResult = await dataLoggingService.ReadAndSaveValuesAsync();
+                    if (valuesResult.BufferTemperatureTop?.Value > 50 && valuesResult.BufferTemperatureTop?.Unit == "°C" && valuesResult.CurrentHeatingPowerDraw?.Value != 0)
+                    {
+                        var schneidControlSettings = _configuration.Get<SchneidControlSettings>();
+                        if (schneidControlSettings?.EmailRecipients?.Count > 0)
+                        {
+                            if (lastEmailSent == null || ((DateTime.UtcNow - lastEmailSent.Value) > TimeSpan.FromHours(12)))
+                            {
+                                var emailSender = scope.ServiceProvider.GetRequiredService<IEmailSender>();
+                                var emailLoggingService = scope.ServiceProvider.GetRequiredService<IEmailLoggingService>();
+                                foreach (var emailRecipient in schneidControlSettings.EmailRecipients)
+                                {
+                                    await emailSender.SendEmailAsync(emailRecipient,
+                                        "Too high temperature.",
+                                        "The temperature is too high and this message is really informative.");
```

Comment:
It's too low😀 I'll add a German message then, you can ignore this part, but please add a TODO for me.